pipeline.py

```diff
@@ -0,0 +1,72 @@
+import pmu,common
+from tabulate import tabulate
+import itertools,sys,re
+
+def pipeline_view(log,depths):
+  out_pipe=open(log.replace('csv','pipeline.log'),'w');
+  pipe_list=get_log(log)
+  event_list=[]
+  depth_list=[]
+  lines=[]
+  for i in depths:
+    event_list.append(depths[i][0])
+    depth_list.append(int(depths[i][1]))
+  issued_name=depths['issued'][0] 
+  issued_list=[i for i,item in enumerate(event_list) if re.search(issued_name,item)]
```

Comment:
Please re-check here if using re is necessary or if you can replace it w/ 'issued_name in item'.

Reply:
fixed